app/models/company_value.rb

```diff
@@ -1,5 +1,5 @@
 class CompanyValue < ApplicationRecord
   validates :title, presence: true
   validates :title, uniqueness: { on: :create, message: 'has already been taken' }
-  validates :title, uniqueness: { on: :create, message: 'has already been taken' }
+  validates :title, uniqueness: { on: :update, message: 'has already been taken' }
```

Comment:
we probably want to have this validation on every occasion so why not use just
`validates :title, uniqueness: true`?